contrail-ui/src/utils/auth-utils.ts

```diff
@@ -18,61 +21,75 @@ export function initializeFirebase() {
         appId: "1:342081308461:web:229362ba93e13630",
     };
     firebase.initializeApp(firebaseConfig);
-    console.log("firebase initialized");
 }
 
 /**
  * Registers a user to Firebase. *NOTE*: {@link initializeFirebase} must be called before.
  * @param displayName the user's display name received from registration
  * @param email the user's email received from registration.
  * @param password the user's password received from registration.
- *
+ * @returns a promise that resolves with the current user, or rejects with the error
  */
 export function registerUser(displayName: string, email: string, password: string) {
```

Comment:
export function registerUser(displayName: string, email: string, password: string) : Promise<${type of user}>

Reply:
done